### PR TITLE
Move from get_logger_config to reading LOGGING from the configuration file in production

### PR DESCRIPTION
**Pull request by cpennington**
get_logger_config was a holdover from edx-platform, and in retrospect, a bad idea. Let's not replicate it with our new IDAs.

### CODE REVIEW

**Comment by cpennington** [comment]
@rlucioni @clintonb: Review?

**Comment by clintonb** [comment]
Is there a related PR to update configuration for devstack/sandboxes/stage/production?

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py [hunk 1]
```diff
@@ -13,115 +13,3 @@ def get_env_setting(setting):
...
-                'level': local_loglevel,
-                'formatter': 'standard',
-                'filename': edx_file_loc,
-                'maxBytes': 1024 * 1024 * 2,
-                'backupCount': 5,
-            },
-        })
-    else:
-        logger_config['handlers'].update({
-            'local': {
```
Comment (rlucioni) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:117]: @cpennington what about these handler updates for development and production environments? Why not include these in local.py and production.py as appropriate?

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py [hunk 2]
```diff
@@ -13,115 +13,3 @@ def get_env_setting(setting):
...
-    "edx_filename" is ignored unless dev_env is set to true since otherwise logging is handled by rsyslogd.
-    """
-
-    # Revert to INFO if an invalid string is passed in
-    if local_loglevel not in ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']:
-        local_loglevel = 'INFO'
-
-    hostname = platform.node().split(".")[0]
-    syslog_format = (
-        "[service_variant={service_variant}]"
```
Comment (rlucioni) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:41]: Why change the log message format? In particular, it seems like including the service name in the message is a good idea.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py [hunk 3]
```diff
@@ -13,115 +13,3 @@ def get_env_setting(setting):
     except KeyError:
         error_msg = "Set the [%s] env variable!" % setting
         raise ImproperlyConfigured(error_msg)
-
-
-def get_logger_config(log_dir='/var/tmp',
-                      logging_env="no_env",
-                      edx_filename="edx.log",
```
Comment (rlucioni) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:20]: Should this filename be retained?

**Comment by cpennington** [comment]
@clintonb: This would need to be changed for IDAs going forward (new IDAs), but I don't think it applies for the cookiecutter changes.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py (hunk 3 above)
Comment (cpennington) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:20]: I think that's going to come from the production syslog configuration.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py (hunk 2 above)
Comment (cpennington) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:41]: This format is for syslog specifically, and should probably come from the production configuration. Although perhaps having good production defaults would be better, so I'll add those to `production.py`.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py (hunk 1 above)
Comment (cpennington) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:117]: The local version is what's used in base, but I'll override it for `production.py`.

Comment (cpennington) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:117]: Oh, sorry. I think this is for our existing vagrant devstack, but for the new docker devstacks, we just need std out.

**Comment by cpennington** [comment]
@rlucioni: I've updated `production.py` with sensible defaults.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py (hunk 3 above)
Comment (rlucioni) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:20]: Is there a PR for this configuration change?

Comment (cpennington) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py:20]: That'll be on a per-service basis.

**Comment by cpennington** [comment]
@clintonb @rlucioni: I jumped the gun. If you need more changes, let me know, and I'll make a new PR.

**Pull request merged**